subprojects/core/src/integTest/groovy/org/gradle/api/tasks/IncrementalInputsIntegrationTest.groovy

```diff
@@ -61,6 +77,22 @@ class IncrementalInputsIntegrationTest extends AbstractIncrementalTasksIntegrati
         return ChangeTypeInternal.ADDED
     }
 
+    @Override
+    String getPrimaryInputAnnotation() {
+        return "@${Incremental.simpleName}"
+    }
```

Comment:
Why is this called the primary input annotation?

Reply:
The annotation is placed on the primary input of the task, the input directory. Do you have a better name for this input property?